_pages/en/esdc-it-strategy.md

```diff
@@ -52,43 +45,114 @@ But a complex and uncertain world rewards an empirical cycle of trying, observin
+The above diagram positions ESDC's IT within its current state of transitioning to Digital Government at the same time as requiring to maintain its legacy systems until they are being replaced with modern ones. In addition, the [4 high level priorities of IITB](http://esdc.prv/en/iitb/corporate/Who_We_Are/Plans_and_Priorities/index.shtml) are listed to remind the focus of the organization.
 
-### Empowering IT teams
+### Input and Outputs
 
-The ESDC IT Strategy uses the [DevOps Research and Assessment (DORA)](https://cloud.google.com/devops) 21 capabilities as a benchmark to identify areas of improvements.
-The choice of technology is not what is important, what is important is to continuously increase frequency of delivery while reducing lead time, time to restore service when an incident occurs, rate of failure, and cyber security incidents.
+**Inputs** are what both the IT Strategy team require to produce their outputs.
+Inputs  for other IITB teams is not displayed here as a means to reduce redundancy and confusion. Other IITB teams play an essential role and have their distinct section (see below)
 
-The transition towards this partnership model is explained in the following [Strategy Map diagram](strategy-summary.html).
-Each block corresponding in the diagram represents a distinct strategy to move ESDC towards its ultimate Digital Transformation goal: become a key stakeholder in the Canadian Government as a Platform.
+**Outputs** are the IT Strategy team's own activities and creation.
+Those outputs are expected to create the short term outcomes listed below, which will lead to the long term impacts sought.
+
+### Short Term Outcomes
+
+As culture is defined by a set of behaviours, the short term outcomes sought by the ESDC IT Strategy is to create the environment needed to manifest them.
+More specifically, an environment that will produce:
+
+1) **Reduced IT project sizes**: Lines of business become more aware of the risks in using technology and so reduce their initiative sizes. Advance planning is shifting to iteration and adjusting course using empirical evidence from user feedback.
+2) **Ongoing efforts visible**: Efforts by IT teams are exposed and more visible by all, promoting a general awareness of IT's complexity and amount of efforts needed to solve problems to stakeholders.
+3) **Challenges surfaced**: Challenges between IT teams, especially relating to information flow and toil, are surfaced to allocate time needed to be address them
+4) **Successes surfaced**: Successes of IT teams are surfaced, communicated within and outside IITB, and celebrated
+5) **Knowledge Shared**: Teams shift from requiring specialists to requiring [t-shaped skills](https://en.wikipedia.org/wiki/T-shaped_skills). Knowledge is easily discoverable and consumable
+6) **Increased User Feedback**: Measurements are build within products and used to inform technical decisions. Quality starts getting validated by end users instead of proxies.
```

Comment:
Capitalization, User and Feedback should be lower case

Reply:
as per your above comment, I've capitalized every first letter of the bold texts in this list instead